runtime-local/src/lib.py

```diff
@@ -75,10 +79,15 @@ def get_services():
     return json.load(open(f"{get_script_path()}/../../runtime.json", encoding="utf-8"))
 
 
-def get_specific_services(service_id: str):
+def get_specific_service(service_id: str):
     """Return the specified service as Python object."""
     services = get_services()
-    return list(filter(lambda service: service['id'] == service_id, services))
+    services = list(filter(lambda service: service['id'] == service_id, services))
+    if len(services) == 0:
+        raise RuntimeError("Service not defined")
```

Comment:
You want the exception message to contain the missing or duplicate `service_id` for better debugging